deploy/infra/deployment.yaml

```diff
@@ -82,6 +82,7 @@ spec:
         args:
           - "--host-ip=$(HOST_IP)"
           - "--node=$(NODE_NAME)"
+          - --metadata-header-required=false
```

Comment:
We can't update the deployment yaml until a new release is made. So can you please revert this change?

Reply:
Sorry for misunderstanding, reverted